installation.md: fix Ansible title level

olivierlemasle opened the pull request:
Documentation on the Ansible role should not be under "Building from Source", as it uses pre-packaged binary.

hitjethva commented:
Hi @lsm5 
I have added step by step guide to install Podman on Ubuntu 20.04 with some basic commands. Please check it and merge pull requests.

lsm5 commented:
@hitjethva thanks, I'll defer to @rhatdan and @TomSweeneyRedHat on addition of posts. LGTM from me fwiw.

TomSweeneyRedHat reviewed (commented):
The post looks fine, thanks for the PR.  I would ask that you create a second file too that will be more permanent.  The New posts we remove after some time (we're actually behind in doing so).  Could you make a second md file, perhaps named:  _posts/2021-06-16-install-podman-on-ubuntu.md with content like:

```
---
title: How to Install and Use Podman on Ubuntu 20.04
layout: default
categories: [blogs]
tags: containers, podman, networking, pod, api, kubernetes, kube, v2, hpc, windows, mac
---
![podman logo](https://podman.io/images/podman.svg)

Hitesh Jethva posted a blog post on the [Atlantic.Net](https://www.atlantic.net/) site talking about [How to Install and Use Podman on Ubuntu 20.04](https://www.atlantic.net/dedicated-server-hosting/how-to-install-and-use-podman-on-ubuntu-20-04/).  In the post Hitesh walks through all the steps necessary from 'A' to 'Z' to get Podman up and running on Ubuntu 20.04 and how to do some initial Podman commands. 

```

TomSweeneyRedHat commented:
Requested additional page supplied in #421

TomSweeneyRedHat commented:
LGTM

olivierlemasle committed:
installation.md: fix Ansible title level

Documentation on the Ansible role should not be under "Building from Source", as it uses pre-packaged binary.

Signed-off-by: Olivier Lemasle <o.lemasle@gmail.com>

TomSweeneyRedHat commented:
@mheon and @lsm5 PTAL

mheon commented:
Sure, LGTM

lsm5 reviewed (approved):
LGTM

lsm5 commented:
I don't have merge rights on this repo. Someone else gotta do it.
